### ISSUE TITLE
Reference moreutils?

### ISSUE THREAD

**Issue by dspinellis** [jlevy/the-art-of-command-line#564]
The [moreutils](https://rentes.github.io/unix/utilities/2015/07/27/moreutils-package/) package includes some interesting command-line utilities.

**Comment by jlevy** [comment]
Yeah, definitely worth a look.  I've not used them but seems like mentioning the key ones would be def worth it. Seems like they are different enough we probably want to mention them individually in the right places, so open to PRs!

Also reminds me, we should mention flock #565
